src/charm.py

```diff
@@ -159,6 +159,15 @@ def external_url(self) -> str:
 
     @property
     def _catalogue_item(self) -> CatalogueItem:
+        api_endpoints = {
+            "Instant query": "/loki/api/v1/query",
+            "Query range": "/loki/api/v1/query_range",
+            "Labels": "/loki/api/v1/labels",
+            "Label values": "/loki/api/v1/label/<name>/values",
+            "Prometheus alerts": "/prometheus/api/v1/alerts",
+            "Prometheus rules": "/prometheus/api/v1/alerts",
```

Comment:
These are not "Prometheus alerts", as this is Loki; this is a Prometheus-compatible endpoint to fetch Loki's alerts, so that some tools that look for Prometheus rules also work with Loki.

Also, this line is duplicated, we should only have one, likely titled "Alerts"